.github/workflows/windows.yml

```diff
@@ -0,0 +1,25 @@
+name: Windows
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: windows-2019
+    steps:
+      - name: Tell Git to use LF
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
+      - uses: actions/checkout@v2
+      - name: Install Cygwin
+        run: choco install -y --no-progress cygwin
+      - name: Install dependencies
+        run: choco install -y --no-progress --source=cygwin git gcc-core make automake pkg-config libiconv-devel libncurses-devel
+      - name: Test Tig
+        run: |
+          $Env:PKG_CONFIG_PATH = "C:\tools\cygwin\lib\pkgconfig"
+          $Env:PATH = "C:\tools\cygwin\bin;" + $Env:PATH
+          make configure
+          sh configure
+          make
+          # make test # TODO There is no TTY on Windows, see https://github.com/actions/runner/issues/241
```

Comment:
Have you tried running `make test` on Windows ? When running in CMD.EXE, there is no TTY anyway, so the script hack may not be necessary for the Cygwin build...

Reply:
yeah, it [fails](https://github.com/krobelus/tig/runs/1676952058?check_suite_focus=true) due to the missing TTY

On a real Windows machine the tests work (but run extremely slowly).

Reply:
Ok then I believe we should drop windows.yml for now and keep relying on the AppVeyor build for Windows CI until there is a solution for this problem. Don't forget to put the AppVeyor badge back in README.adoc and I think we are good to go.

Reply:
ok, done